sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/LinkedEntityResult.java

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics.models;
+
+import com.azure.core.annotation.Fluent;
+
+import java.util.List;
+
+
+/**
+ * The LinkedEntityResult model.
+ */
+@Fluent
```

Comment:
Should this be immutable?

Reply:
We are trying to get a review on the DetectedLanguageResult first. Other operations, such as NamedEntity,  PII Entity, KeyPhrases, and Sentiment has been check-in.  Trying to split above operation into difference PR but seems it might need to goes here.  So thank for reviewing it now.

Reply:
I'm saying the annotation is wrong. There are no setter methods here.